addons/60fps/addon.json

```
@@ -1,10 +1,15 @@
 {
-  "name": "60FPS project player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom project framerate",
```

Custom or customizable sounds kind of like you can set it to whatever you want whenever you want, and, I mean, you can kind of, you can go into the addon settings and change it, but you can only _quickly_ switch between 30 FPS and one custom framerate, so maybe the name shouldn't over-express the customizability aspect.
```suggestion
  "name": "Higher project framerate",
```

---

Anyone have opinions on what the name should be? It's true that it's customizable, but whatever custom framerate you've set needs to be adjusted through the settings, and also requires a gesture to activate.